### PR TITLE
chore: add rc-dock to recommendation pages

### PR DESCRIPTION
**Pull request by rinick**
### 🤔 This is a ...

- [x] Site / documentation update

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/29719

### 💡 Background and solution

将rc-dock添加到社区精选组件的推荐页面

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | add rc-dock to recommendation pages |
| 🇨🇳 Chinese | 在社区精选组件页面加入rc-dock |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: docs/react/recommendation.zh-CN.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ title: 社区精选组件
 | 路由 | [react-router](https://github.com/ReactTraining/react-router) |
 | 布局 | [@rebass/grid](https://github.com/rebassjs/grid) [react-blocks](https://github.com/whoisandy/react-blocks) [react-flexbox-grid](https://github.com/roylee0704/react-flexbox-grid) |
 | 拖拽 | [react-beautiful-dnd](https://github.com/atlassian/react-beautiful-dnd/) [react-dnd](https://github.com/gaearon/react-dnd) [react-sortable-hoc](https://github.com/clauderic/react-sortable-hoc) |
+| 可拖拽窗口 | [rc-dock](https://github.com/ticlo/rc-dock) |
```
Comment (afc163) [inline docs/react/recommendation.zh-CN.md:15]: 放在布局里吧，布局里几个库都很久不维护了，都删了再补上 https://github.com/react-grid-layout/react-grid-layout https://github.com/sealninja/react-grid-system/

Comment (rinick) [reply docs/react/recommendation.zh-CN.md]: > 放在布局里吧，布局里几个库都很久不维护了，都删了再补上 https://github.com/react-grid-layout/react-grid-layout https://github.com/sealninja/react-grid-system/

OK

File: docs/react/recommendation.en-US.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ title: Third-Party Libraries
 | Visualization and charts | [AntV Data Visualization](https://antv.vision/en) [🔥 AntV Charting Library](https://g2plot.antv.vision/en) [BizCharts](https://github.com/alibaba/BizCharts) [recharts](https://github.com/recharts/recharts/) [viser](https://viserjs.github.io/) |
 | React Hooks Library | [ahooks](https://github.com/alibaba/hooks) |
 | Router | [react-router](https://github.com/ReactTraining/react-router) |
-| Layout | [@rebass/grid](https://github.com/rebassjs/grid) [react-blocks](http://whoisandy.github.io/react-blocks/) [react-flexbox-grid](https://github.com/roylee0704/react-flexbox-grid) |
+| Layout | [rc-dock](https://github.com/ticlo/rc-dock) [react-grid-layout](https://github.com/react-grid-layout/react-grid-layout) [react-grid-system](https://github.com/sealninja/react-grid-system) |
```
Comment (afc163) [inline docs/react/recommendation.en-US.md:13]: rc-dock 放在最后吧

Comment (rinick) [reply docs/react/recommendation.en-US.md]: 好吧

**Pull request merged**